Source/WebCore/platform/cocoa/ParentalControlsURLFilter.mm

```diff
@@ -82,13 +81,28 @@ static bool wcrBrowserEngineClientEnabled()
     return [PAL::getWCRBrowserEngineClientClassSingleton() shouldEvaluateURLs];
 }
 
-ParentalControlsURLFilter& ParentalControlsURLFilter::singleton()
+static RefPtr<ParentalControlsURLFilter>& globalFilter()
 {
-    static MainThreadNeverDestroyed<UniqueRef<ParentalControlsURLFilter>> filter = UniqueRef(*new ParentalControlsURLFilter);
+    static NeverDestroyed<RefPtr<ParentalControlsURLFilter>> filter;
     return filter.get();
 }
 
-ParentalControlsURLFilter::ParentalControlsURLFilter() = default;
+Ref<ParentalControlsURLFilter> ParentalControlsURLFilter::sharedFilter()
```

Comment:
Ditto, why does this return a `Ref<>` and not a `ParentalControlsURLFilter&`, like `singleton()` used to return. We're not transferring ownership here.